docs/tools/chrome-extension/refactoring-and-testing.md

```diff
@@ -1,16 +1,20 @@
 ---
 id: refactoring-and-testing
 title: 'Refactoring and Testing'
-sidebar_label: 'Refactoring and Testing'
+sidebar_label: Refactoring and Testing
 keywords:
   - refactoring
-  - testing 
+  - testing
 ---
 
+## Refactoring a Block of Code
+
 Refactor a block of code by using the blue plus button that is displayed when hovering over a line. To select a block of code, hover over the starting line, click and drag the blue plus button until the desired line.
 
 ![generate code refactor](../../../static/gif/pr-code-refactor.gif)
 
+## Testing a Block of Code
+
 Generate tests for a block of code by using the blue plus button that is displayed when hovering over a line. To select a block of code, hover over the starting line, click and drag the blue plus button until the desired line.
 
 ![generate code refactor](../../../static/gif/pr-code-test.gif)
```

Comment:
Like the [feedback I wrote for code explanation](https://github.com/open-sauced/docs/pull/213#discussion_r1405460110), I can't test this out because the "Refactor Code" and "Test Code" don't work for me.

The same feedback as before, we need to update the explanation. One of the reasons is there is no explanation about clicking the "orange OpenSauced" button to generate the code refactor and test.